client/influxdb.go

```diff
@@ -7,20 +7,66 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"net"
 	"net/http"
 	"net/url"
+	"strconv"
+	"strings"
 	"time"
 
 	"github.com/influxdb/influxdb/influxql"
 	"github.com/influxdb/influxdb/tsdb"
 )
 
+const (
+	// DEFAULT_HOST is the default host used to connect to an InfluxDB instance
+	DEFAULT_HOST = "localhost"
+	// DEFAULT_PORT is the default port used to connect to an InfluxDB instance
+	DEFAULT_PORT = 8086
+	// DEFAULT_TIMEOUT is the default connection timeout used to connect to an InfluxDB instance
+	DEFAULT_TIMEOUT = 0
+)
+
 // Query is used to send a command to the server. Both Command and Database are required.
 type Query struct {
 	Command  string
 	Database string
 }
 
+// ParseConnectionString will parse a string to create a valid connection URL
+func ParseConnectionString(path string, ssl bool) (url.URL, error) {
```

Comment:
Could the `path` just require `http://...` or `https://...` and then no need for `ssl bool`?